### PR TITLE
rosegarden package was added

### PR DESCRIPTION
**Pull request by lebastr**

### CODE REVIEW

File: pkgs/applications/audio/rosegarden/default.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+let rosegarden = rec {
+  version = "14.12";
+  base_name = "rosegarden";
+  name = "${base_name}-${version}";
+  url  = "mirror://sourceforge/rosegarden/${name}.tar.bz2";
+  sha256 = "0zhlxr1njyy6837f09l6p75js0j5mxmls6m02bqafv9j32wgnxpq";
+
+  homepage = http://www.rosegardenmusic.com/;
+  description = "Rosegarden is a music composition and editing environment.";
```
Comment (jagajaga) [inline pkgs/applications/audio/rosegarden/default.nix]: Please avoid pkg's name in the description. And remove dot at the end.

**Comment by jagajaga** [comment]
Please rename commit's name following our guide in `CONTRIBUTING.md`

**Comment by lebastr** [comment]
Replaced by #9450

**Comment by jagajaga** [comment]
Don't open new PR, you can update the existing one with `git push --force`

**Pull request closed without merging**